CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 0.12.1-dev
+
```

Comment:
Need an entry?

Reply:
Non-user visible change. I think not...

Reply:
I guess the removal of the `async` dep is technically user facing... In general for changes which are not user facing our new approach is to require the section header, but not necessarily an entry.

https://github.com/dart-lang/sdk/wiki/External-Package-Maintenance